src/CorrugatedIron.Tests/Models/MapReduce/RiakMapReduceTests.cs

```diff
@@ -38,6 +40,9 @@ public class RiakMapReduceTests
         private const string ComplexMrJobWithFilterText =
             @"{""inputs"":{""bucket"":""animals"",""key_filters"":[[""matches"",""spider""]]},""query"":[{""map"":{""language"":""javascript"",""source"":""function(o) { return [1]; }"",""keep"":false}},{""reduce"":{""language"":""javascript"",""name"":""Riak.reduceSum"",""keep"":true}}]}";
 
+        private const string ComplexMrJobWithTypeAndFilterText =
+            @"{""inputs"":{""bucket"":[""zoo"",""animals""],""key_filters"":[[""matches"",""spider""]]},""query"":[{""map"":{""language"":""javascript"",""source"":""function(o) { return [1]; }"",""keep"":false}},{""reduce"":{""language"":""javascript"",""name"":""Riak.reduceSum"",""keep"":true}}]}";
```

Comment:
I really don't know what I like better. `@""` strings or `\` abuse. I guess another option would be a resources file.